backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/services/CoreFeaturesSwitch.java

```diff
@@ -59,6 +61,14 @@ public List<EStructuralFeature> defaultCase(EObject object) {
         return List.of();
     }
 
+    @Override
```

Comment:
Please put it at the appropriate location (all cases are ordered alphabetically)